src/halomod/halo_model.py

```diff
@@ -64,7 +64,7 @@ def __init__(
         exclusion_model="NoExclusion",
         exclusion_params=None,
         colossus_params=None,
-        hc_spectrum="nonlinear",
+        hc_spectrum="linear",
```

Comment:
While I *think* I agree that this should be changed, we need to be a little careful, because changing a default breaks the API (i.e. we'd need to bump a major version). This is fine, and we can certainly do it, but it should be in a separate PR where we can conglomerate some API-breaking changes (so we don't shoot up to v15 too quickly). It also needs to be specifically mentioned in the CHANGELOG.

Reply:
ok I don't think I fully understand what's going on but I get this causes problem. I'll leave this up here and create a separate PR for cross-correlation